_events/repeated/community-call.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Community Call
+subtitle:
+location: Virtual
+expires: 2025-12-11
+event_date: "second Thursday each month 12:00-13:00 Eastern"
+layout: event
+time:
+  - - start 2022-01-13
+
+# Repeated events information
+repeated: true
+
+# use an rdate string instead (best for complex repeated events)
+# note that the dtstart and rdate at the end are the same
+rrule: 
+  - DTSTART;TZID=America/New_York:20220113T120000
+# first tuesday of every month
```

Comment:
This stuff is hard, you did great! Minor tweak here.
```suggestion
# second Thursday of every month
```

Reply:
👍